.devcontainer/devcontainer.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "TypeScript Website codespace",
+  "name": "Lingua Franca Website codespace",
   "dockerFile": "Dockerfile",
-
   // Set *default* container specific settings.json values on container create.
   "settings": {
     "terminal.integrated.shell.linux": "/bin/bash"
   },
-
-  "forwardPorts": [8000],
-
+  "forwardPorts": [
+    8000
+  ],
   // Use 'postCreateCommand' to run commands after the container is created.
   "postCreateCommand": "sudo yarn install && sudo yarn bootstrap && yarn start"
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Locally, when I do `cat`, I don't see evidence of a missing terminating newline. I think that GitHub uses a special marker to show when there is no terminating newline:
![image](https://user-images.githubusercontent.com/33707478/174677578-07d215cc-708a-486f-b49f-ef04177ee81b.png)

This rendering differs from the behavior e.g. in VS Code for the same underlying text.